[CI] InferenceRestIT test {p0=inference/30_semantic_text_inference/Calculates embeddings using the default ELSER 2 endpoint} failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #11966 / ubuntu-2404-aarch64_checkpart2_platform-support-arm](https://gradle-enterprise.elastic.co/s/tcvanmladysro)
- [elasticsearch-periodic-platform-support #11966 / almalinux-8-aarch64_checkpart2_platform-support-arm](https://gradle-enterprise.elastic.co/s/ci2ffcwzbioi2)
- [elasticsearch-periodic-platform-support #11966 / windows-2022_checkpart2_platform-support-windows](https://gradle-enterprise.elastic.co/s/qsj5lduko54vi)
- [elasticsearch-periodic-platform-support #11966 / windows-2025_checkpart2_platform-support-windows](https://gradle-enterprise.elastic.co/s/snn5zrte7tdsm)
- [elasticsearch-periodic-platform-support #11966 / ubuntu-2404-aarch64_checkpart2_platform-support-arm](https://gradle-enterprise.elastic.co/s/pwfk2buzx2iu2)
- [elasticsearch-periodic-platform-support #11966 / almalinux-8-aarch64_checkpart2_platform-support-arm](https://gradle-enterprise.elastic.co/s/dmelyudzvh3e4)
- [elasticsearch-pull-request #128072 / part-2](https://gradle-enterprise.elastic.co/s/mpioi7idaomg4)
- [elasticsearch-pull-request #128071 / part-2](https://gradle-enterprise.elastic.co/s/5m56fskc3q4va)
- [elasticsearch-pull-request #128065 / part-2](https://gradle-enterprise.elastic.co/s/rukhxbutjof4o)
- [elasticsearch-pull-request #128069 / part-2](https://gradle-enterprise.elastic.co/s/svfbgojyrwp4g)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:inference:yamlRestTest" --tests "org.elasticsearch.xpack.inference.InferenceRestIT.test {p0=inference/30_semantic_text_inference/Calculates embeddings using the default ELSER 2 endpoint}" -Dtests.seed=BEA439C83DADBAF5 -Dtests.locale=ia -Dtests.timezone=Pacific/Wake -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.inference.InferenceRestIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Bp0%3Dinference%2F30_semantic_text_inference%2FCalculates%20embeddings%20using%20the%20default%20ELSER%202%20endpoint%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: org.junit.internal.runners.model.MultipleFailureException: There were 7 errors:
  java.lang.AssertionError(expected [2xx] status code but api [index] returned [409 Conflict] [{"error":{"root_cause":[{"type":"status_exception","reason":"Error in inference process: [inference canceled as process is stopping]","stack_trace":"org.elasticsearch.ElasticsearchStatusException: Error in inference process: [inference canceled as process is stopping]\n\tat org.elasticsearch.ml@9.4.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.deployment.AbstractPyTorchAction.onFailure(AbstractPyTorchAction.java:122)\n\tat org.elasticsearch.ml@9.4.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.deployment.InferencePyTorchAction.processResult(InferencePyTorchAction.java:181)\n\tat org.elasticsearch.ml@9.4.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.deployment.InferencePyTorchAction.lambda$doRun$3(InferencePyTorchAction.java:150)\n\tat org.elasticsearch.server@9.4.0-SNAPSHOT/org.elast
[truncated]
```

**Issue Reasons:**
- [main] 4 consecutive failures in test test {p0=inference/30_semantic_text_inference/Calculates embeddings using the default ELSER 2 endpoint}
- [main] 2 consecutive failures in step ubuntu-2404-aarch64_checkpart2_platform-support-arm
- [main] 2 consecutive failures in step almalinux-8-aarch64_checkpart2_platform-support-arm
- [main] 11 failures in test test {p0=inference/30_semantic_text_inference/Calculates embeddings using the default ELSER 2 endpoint} (3.4% fail rate in 322 executions)
- [main] 2 failures in step ubuntu-2404-aarch64_checkpart2_platform-support-arm (100.0% fail rate in 2 executions)
- [main] 2 failures in step almalinux-8-aarch64_checkpart2_platform-support-arm (100.0% fail rate in 2 executions)
- [main] 5 failures in step part-2 (2.2% fail rate in 230 executions)
- [main] 5 failures in pipeline elasticsearch-pull-request (2.2% fail rate in 231 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -303,18 +303,6 @@ tests:
 - class: org.elasticsearch.xpack.inference.integration.AuthorizationTaskExecutorIT
   method: testCreatesChatCompletion_AndThenCreatesTextEmbedding
   issue: https://github.com/elastic/elasticsearch/issues/138012
-- class: org.elasticsearch.xpack.inference.InferenceRestIT
-  method: test {p0=inference/30_semantic_text_inference/Calculates embeddings using the default ELSER 2 endpoint}
-  issue: https://github.com/elastic/elasticsearch/issues/144159
-- class: org.elasticsearch.xpack.inference.InferenceRestIT
-  method: test {p0=inference/30_semantic_text_inference_bwc/Calculates embeddings using the default ELSER 2 endpoint}
-  issue: https://github.com/elastic/elasticsearch/issues/144160
-- class: org.elasticsearch.xpack.inference.InferenceRestIT
-  method: test {p0=inference/40_semantic_text_query/Query a field that uses the default ELSER 2 endpoint}
-  issue: https://github.com/elastic/elasticsearch/issues/144161
-- class: org.elasticsearch.xpack.inference.DefaultEndPointsIT
-  method: testInferDeploysDefaultRerank
-  issue: https://github.com/elastic/elasticsearch/issues/144162
 - class: org.elasticsearch.upgrades.MappingSupplementaryCharacterRollingUpgradeIT
   method: testMappingWithSupplementaryCharacterFieldName {upgradedNodes=3}
   issue: https://github.com/elastic/elasticsearch/issues/144163

